executable/microsoft_pe.ksy

```diff
@@ -145,6 +145,7 @@ types:
       name_zeroes:
         pos: 0
         type: u4
+        if: name_zeroes == 0
```

Comment:
This by itself doesn't make much sense. Basically, if compiles into an infinite recursion, i.e. to get value of `name_zeroes`, one must check if expression, and for that we need to get value of `name_zeroes`.